source/code/include/playfab/PlayFabIXHR2HttpPlugin.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <stdafx.h>
```

Comment:
I just realized I made these changes in the header. I don't think your other headers do this. my bad. I think this actually belongs in the associated cpp files

Reply:
figured out why this was needed. it's because of a problem in the xbox vcxproj file again. I'll highlight the problem there for ease